docs/authentication/social-connections/account-linking.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking and manages unverified email addresses from OAuth providers.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both OAuth and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an OAuth provider returns an email address that matches an existing account, assuming a single owner for each email address.
+
+## How it works
+
+When a user attempts to sign-in or sign-up, Clerk validates the email address provided. If the email comes from an OAuth provider, there are several scenarios depending on the verification status of the email and whether an account with the same email already exists. Below are the different scenarios and how Clerk handles them.
```

Comment:
:question: Should the verbs be "sign in" and "sign up"? Without the hyphen.

Suggestion
```
When a user attempts to sign in or sign up, Clerk checks the provided email address. If the email address comes from an OAuth provider, depending on the email address verification status and whether an account with the same email address exists, Clerk will try to link the email address with the existing account. 

Let's look at the different possibilities and how Clerk handles each of them.
```

Reply:
✅ Added a bit more context, plus your suggestions